p4-16/psa/PSA.mdk

```diff
@@ -421,41 +447,40 @@ error to instantiate the `clone` extern in the P4 control or parser
 block.
 
 A PSA implementation provides two configuration metadata, `clone`
-and `clone_spec`, to the PRE to control the cloning mechanism. The
-`clone` enables/disables the cloning mechanism. If the `clone`
-bit is set, the clone mechanism generates a cloned packet with the
-optional attached metadata. If the `clone` bit is unset or
-uninitialized, the clone mechanism is disabled and a cloned packet is
+and `clone_port`, to the PRE to control the cloning mechanism. When
+the `clone` bit is set, the clone mechanism generates a cloned packet with the
+optional attached metadata. When the `clone` bit is unset or
+uninitialized, cloning is disabled and a cloned packet is
```

Comment:
Elsewhere in the PSA draft the `clone` bool cannot be uninitialized, as it is defined to be initialized to `false` at the beginning of the ingress and egress control blocks.